scripts/main.js

```diff
@@ -0,0 +1,111 @@
+import sendRequest from './reserve.js';
+
+const formBudgetedExpenseYes = document.querySelector('.intro #budgeted-expense-yes');
+const formBudgetedExpenseNo = document.querySelector('.intro #budgeted-expense-no');
+const formOfficerName = document.querySelector('.intro #officer-name');
+const formOfficerPosition = document.querySelector('.intro #officer-position');
+const formDirectDeposit = document.querySelector('.intro #direct-deposit');
+const formCheck = document.querySelector('.intro #check');
+const formAddress = document.querySelector('.intro #address');
+const formReceiptImage = document.querySelector('.intro #receipt-image');
+const formDocumentsFile = document.querySelector('.intro #supporting-documents');
+const formSubmitButton = document.querySelector('.intro #submit-button');
+
+const name = document.getElementById('name');
+const position = document.getElementById('position');
+const email = document.getElementById('email');
+const mId = document.getElementById('mId');
+const date = document.getElementById('date');
+const vendor = document.getElementById('vendor');
+const amount = document.getElementById('amount');
+const description = document.getElementById('description');
+const officerName = document.getElementById('officer-name');
+const officerPosition = document.getElementById('officer-position');
+const directDeposit = document.getElementById('direct-deposit');
+const check = document.getElementById('check');
+const address = document.getElementById('address');
+const receiptImage = document.getElementById('receipt-image');
+const supportingDocuments = document.getElementById('supporting-documents');
+
+const xmlHttp = new XMLHttpRequest();
```

Comment:
Why are using `XMLHttpRequest` here but `fetch` somewhere else? You should keep it consistent. I suggest you to use Fetch as it's a newer API and it's something we use across the rest of our web apps.